### PR TITLE
feat: update sentry

### PR DESCRIPTION
**Pull request by aon**
Update Sentry to latest version, following bug detected in https://github.com/epicweb-dev/epic-stack/issues/744 and bug reported (and fixed) in Sentry:

- https://github.com/getsentry/sentry-javascript/issues/12285

## Checklist

- [x] Tests updated
- [x] Docs updated

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -19,9 +18,9 @@ const IS_PROD = MODE === 'production'
 const IS_DEV = MODE === 'development'
 const ALLOW_INDEXING = process.env.ALLOW_INDEXING !== 'false'
 
-const createRequestHandler = IS_PROD
-	? Sentry.wrapExpressCreateRequestHandler(_createRequestHandler)
-	: _createRequestHandler
+if (IS_PROD && process.env.SENTRY_DSN) {
+	import('../app/utils/monitoring.server.ts').then(({ init }) => init())
```
Comment (kentcdodds) [inline server/index.ts:22]: This won't work when deployed because this file isn't bundled. Can we move the monitoring file to the server directory?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thank you!

File: server/index.ts (hunk 1 above)
Comment (aon) [reply server/index.ts]: Fixed!

**Review by kentcdodds — APPROVED** [review]
Solid. Thank you!

**Pull request merged**